### PR TITLE
Category list page - BW-559

### PR DESCRIPTION
**Pull request by dgmdan**
Implements a "use cuisine types" option. If that option is not checked, the admin can create a custom list of categories to display. Resolves BW-559.

### CODE REVIEW

File: Bizyhood/Public/css/broadstreet.css [hunk 1]
```diff
@@ -237,3 +237,12 @@ select#zip_codes {
     width:80px;
     height:100px;
 }
+
```
Comment (eugenebiz) [inline Bizyhood/Public/css/broadstreet.css]: Let's add a new stylesheet for BH specific tweaks.

File: Bizyhood/Public/js/broadstreet.js [hunk 2]
```diff
@@ -55,6 +55,41 @@ jQuery(function($){
         // Clear ZIP code field, put focus on it
         $('#zip_code').val('').focus();
     });
+
+    $('#category_add').click(function() {
```
Comment (eugenebiz) [inline Bizyhood/Public/js/broadstreet.js]: Same as stylesheet, let's get a separate javascript source file going for BH specific stuff.

Comment (dgmdan) [inline Bizyhood/Public/js/broadstreet.js]: I was going to rename these to bizyhood.js and bizyhood.css in a later
story, would that be ok? I don't think we'd need both.

Comment (eugenebiz) [inline Bizyhood/Public/js/broadstreet.js]: Sure, that'll work. Any specific reason to wait?

Comment (dgmdan) [inline Bizyhood/Public/js/broadstreet.js]: Planning on doing that in BW-561

**Comment by eugenebiz** [comment]
Ship it.

**Pull request merged**